configure.ac

```diff
@@ -205,6 +205,8 @@ AX_GCC_FUNC_ATTRIBUTE([format])
 AX_TYPE_SOCKLEN_T
 AX_CFLAGS_WARN_ALL
 AX_APPEND_COMPILE_FLAGS([-Wwrite-strings])
+AX_APPEND_COMPILE_FLAGS([-Wmissing-prototypes], ,[-Werror])
+AX_APPEND_COMPILE_FLAGS([-Wno-error=missing-prototypes], ,[-Werror])
```

Comment:
Don't understand the 2nd line here. What is this doing?

Reply:
That tells the compiler not to treat that warning as an error as I still haven't fixed all of them. As it is just two functions I might still do it in this PR.

BTW that is also like the remaining compiler warnings on BSD should be handled a few lines below to make sure none of a different type are being introduced.

Reply:
If it's being removed later, I'm fine with that - thanks.

Reply:
Yeah. Any such things are (hopefully) temporary. Such things are always fidgety if you apply them to a "legacy" code base.

Reply:
These lines can be removed now I think.

Reply:
Still required until the latest `libpainter` and `librfxcodec` are being used.

Reply:
Those changes have been merged.

You can add a commit to your PR which pulls in the latest versions:-
```sh
cd libpainter
git checkout devel
git pull
cd ../librfxcodec
git checkout devel
git pull
cd ..
git add libpainter librfxcodec
git commit -m "Update submodules for -Wmissing-prototypes"
```

Or something like that. You get the idea.